specs/protocol/superchain-config.md

```diff
@@ -5,60 +5,243 @@
-a single Superchain network.
+The SuperchainConfig contract is used to manage global configuration values for multiple OP Chains
+within a single Superchain network.
 
-## Configurable values
+## Definitions
 
-The `SuperchainConfig` contract manages the following configuration values:
+### Guardian
 
-- `PAUSED_SLOT`: A boolean value indicating whether the Superchain is paused.
-- `GUARDIAN_SLOT`: The address of the guardian, which can pause and unpause the system.
+The **Guardian** is a dedicated role that in the OP Stack that is permitted to trigger certain
+actions to maintain the security of an OP Chain or set of OP Chains in case of a bug in the
+protocol. In the Superchain, the Guardian role is held by the Optimism Security Council.
 
-## Configuration data flow
+### Pause Deputy
 
-All contracts which read from the `SuperchainConfig` contract hold its address as storage values
-in the proxy account, and call directly to it when reading configuration data.
+The **Pause Deputy** is a dedicated role managed by the [Guardian](#guardian) that can execute the
+[Pause Mechanism](#pause-mechanism). An OP Chain does not necessarily need to assign a Pause
+Deputy. The Pause Deputy is capable of triggering the Pause Mechanism but does not have the ability
+to reset the mechanism or unpause the system.
 
-```mermaid
-flowchart TD
-StandardBridge --> SuperchainConfig
-L1ERC721Bridge --> SuperchainConfig
-L1CrossDomainMessenger --> SuperchainConfig
-OptimismPortal --> SuperchainConfig
-```
+The Pause Deputy is an optional role within an OP Chain and can be configured if the Guardian is
+a [Safe][safe-docs] that installs the [Deputy Pause Module](./deputy-pause-module.md).
 
-### Pausability
+### Pause Identifier
 
-The Superchain pause feature is a safety mechanism designed to temporarily halt withdrawals from the system in
-an emergency situation. The Guardian role is authorized to pause and unpause the system.
+The **Pause Identifier** is an address parameter used to specify the scope of a pause action. This
+identifier determines which systems or chains are affected by the pause:
 
-#### Paused identifiers
+- When the identifier is the zero address (`0x0000000000000000000000000000000000000000`), the pause
+  applies globally to all chains sharing the `SuperchainConfig` contract.
+- When the identifier is a non-zero address, the pause applies specifically to the chain or set of
+  chains associated with that identifier.
 
-When the system is paused the `Paused(string identifier)` event is emitted. This allows for the
-caller to provide additional information to be used during incident response.
+The identifier must be an `ETHLockbox` address or `address(0)`. This allows for targeted pausing of
+either specific chains, the interop set (which shares an `ETHLockbox` contract), or the all chains
+that share the same `SuperchainConfig` when `address(0)` is used as the identifier.
```

Comment:
Does pausing revert if the identifier is an unknown address? e.g. do we call a method on the identifier to verify it's likely a lockbox?

Reply:
For simplicity I figured we wouldn't, we maybe could? I'm not strongly opinionated here.

Reply:
Reverting is nice since it's a little safer, but I think we can only probabilistically revert (i.e. I think we can't know for certainty if an indentifier is actually an ETH Lockbox that's part of an L2 using this superchain config?). Checking bytecode feels excessive, so the best we can do is call a method like `paused` or `proxyAdminOwner` to check if it's likely the address is a lockbox at all

Reply:
I'd like to leave this as-is for now, I'm worried that a revert here adds risk without major benefit, especially if there are contracts on different versions that have different functions. We can keep legacy functions in place but I'm not confident that explicitly limiting this to ETHLockbox addresses gives us any significant benefit.